doc/source/serve/production-guide/fault-tolerance.md

```diff
@@ -592,7 +592,7 @@ $ python
 
 ### HTTPProxy failure
```

Comment:
Maybe update this to just Proxy as well

Reply:
done - I also updated the sentence underneath to state Proxy only